From 22bd5ef5c2eb2ce14d64c8952a03c5548b91b704 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Tue, 8 Mar 2022 18:02:24 +0100
Subject: [PATCH 341/529] ARM: configs: enable CONFIG_MEDIA_GC2145

Enable the GC2145 camera sensor module used on the stm32mp135f-dk

Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Change-Id: Ifd46999855c8097dd90720844be13eb441fe7128
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/282545
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Hugues FRUCHET <hugues.fruchet@st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -65,6 +65,7 @@ CONFIG_UCSI_STM32G0=m
 CONFIG_USB_ONBOARD_HUB=y
 CONFIG_USELIB=y
 CONFIG_VFP=y
+CONFIG_VIDEO_GC2145=m
 CONFIG_VIDEO_OV5640=m
 CONFIG_VIDEO_ST_MIPID02=m
 CONFIG_ZBOOT_ROM_BSS=0x0
